packages/Statie/src/Console/Command/InitCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\Statie\Console\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Filesystem;
+use Symplify\PackageBuilder\Console\Command\CommandNaming;
+use Symplify\PackageBuilder\Console\ShellCode;
+use function Safe\getcwd;
+
+final class InitCommand extends Command
+{
+    /**
+     * @var SymfonyStyle
+     */
+    private $symfonyStyle;
+
+    /**
+     * @var Filesystem
+     */
+    private $filesystem;
+
+    public function __construct(SymfonyStyle $symfonyStyle, Filesystem $filesystem)
+    {
+        parent::__construct();
+        $this->symfonyStyle = $symfonyStyle;
+        $this->filesystem = $filesystem;
+    }
+
+    protected function configure(): void
+    {
+        $this->setName(CommandNaming::classToName(self::class));
+        $this->setDescription('Generate a bare site.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->filesystem->mirror(__DIR__ . '/../../../templates/statie-website', getcwd());
+
+        $this->symfonyStyle->success('Your new Statie is now generated');
+
+        $this->symfonyStyle->note('Run "npm install" to get javascript packages');
+        $this->symfonyStyle->note('Then run "gulp" to run website in your browser "localhost:8000"');
+
+        $this->symfonyStyle->note('Do you blog? Keep "/source/_posts" directory. If not, you can drop it');
```

Comment:
Is this message necessary? If someone doesn't blog, they do not need `_layouts/post.twig` as well. I'd rather skip this message altogether.

Reply:
It should prevent "WTF is post? where is more code I don't need? Oh no". This command is dumping more files, so this the least we can do.

I'll add the "_layouts/post.twig" note as well :+1:

Reply:
Or maybe `--no-blog` option?

Reply:
To keep it simple I think the note about layout is sufficient 😄

Reply:
I forgot there also option to simply ask  :)